README.md

```diff
@@ -33,7 +33,7 @@ The Azure DevOps Migration Tools allow you to bulk edit and migrate data between
 ## Quick Links
 
  - [Video Overview](https://youtu.be/ZxDktQae10M)
- - [Getting Started](http://nkdagility.github.io/azure-devops-migration-tools/getting-started.md)
```

Comment:
Since this document is used in locations other than on github this needs to be a full link to the documentation.

Reply:
i've reverted my changes to that link